docs/preview/features/sinks/azure-application-insights.md

```diff
@@ -41,6 +41,45 @@ ILogger logger = new LoggerConfiguration()
     .CreateLogger();
 ```
 
+## Configuration
+
+The Azure Application Insights sink has some additional configuration which can be changed to influence the tracking.
+
+### Exceptions
+
+#### Property format
+
+When tracking exceptions, all the public properties of the exception will be included as [custom dimensions](https://docs.microsoft.com/en-us/azure/azure-monitor/app/api-custom-events-metrics#custom-measurements-and-properties-in-analytics).
+These public properties are formatted with the following pattern: `Exception-{0}` where `{0}` is the place where the public property's name is inserted. 
+The value of the property will be the value of the custom dimension so that the custom dimension will be in the form `"Exception-{your-property-name}" = "your-property-value"`.
+
+This property format pattern can be configured, like shown in the following example:
+
+```csharp
+using Serilog;
+using Serilog.Configuration;
+
+ILogger logger = new LoggerConfiguration()
+    .WriteTo.AzureApplicationInsights("<key>", options => options.Exception.PropertyFormat = "CustomException.{0}")
```

Comment:
Maybe it might be good to have this as an opt-out/opt-in? Opt-in might be safer as this can reduce decent chunk of properties.

Reply:
Yes, I'll agree. Let's do that!